QModManager/Patching/ManifestValidator.cs

```diff
@@ -14,12 +14,12 @@ internal class ManifestValidator
     {
         internal static readonly Regex VersionRegex = new Regex(@"(((\d+)\.?)+)");
 
-        internal static readonly Dictionary<string, string> BannedIDs = new Dictionary<string, string>()
+        internal static readonly Dictionary<string, Tuple<string, ModStatus>> BannedIDs = new Dictionary<string, Tuple<string, ModStatus>>()
         {
-            { "QModManager", "This mod ID is banned. Please pick another one." },
-            { "QModInstaller", "This mod ID is banned. Please pick another one." },
+            { "QModManager", new Tuple<string, ModStatus>("This mod ID is banned. Please pick another one.", ModStatus.InvalidCoreInfo) },
+            { "QModInstaller", new Tuple<string, ModStatus>("This mod ID is banned. Please pick another one.", ModStatus.InvalidCoreInfo) },
             // The list can be continued in the future with stuff like
-            // { "RadialTabs", "Mod merged with QModManager" },
+            // { "RadialTabs", new Tuple<string, ModStatus>("Mod merged with QModManager", ModStatus.CanceledByUser) },
```

Comment:
This got really messy really fast.  
Each `ModStatus` should correlate to one and only one status.

If you want to handle more messages, add more entries to `ModStatus`.  
Update the `SummaryLogger` accordingly.

Reply:
The idea behind it is that some mods should throw an error when this happens, while others should be silently ignored.

Reply:
As discussed over Discord,  
update the `ModStatus` enum to include any new scenarios we are handling. 

We should not overload a single `ModStatus` value with more than one meaning.  
So instead we can expand this enum to cover the new scenarios of:
- Prohibited ModId value
- Mod that has been incorporated into QMM, has not been patched, and can be safely removed from the QMod folder